recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,357 @@
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+  
+  # s2v_packages
+  {% if use_s2v_packages is defined %}
+  {% if use_s2v_packages or use_all_subpackages %}
+  {% with subpackage_id = "sense2vec" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # s2v_packages = ["sense2vec>=1.0.2"] + spacy_packages
+        # "sense2vec": base_packages + s2v_packages
+        - {{ pin_subpackage(name_ ~ "-spacy", exact=True) }}
+        - sense2vec {{ libver_sense2vec }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+
+  ### tf_packages
+  {% if use_tf_packages is defined %}
+  {% if use_tf_packages or use_all_subpackages %}
+  {% with subpackage_id = "tfhub" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+    # NOTE: this throws error for windows build.
+      skip: True  # [win]
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "tfhub": base_packages + tf_packages
+        - {{ name_ }} {{ version }}
+        - tensorflow {{ libver_tensorflow }}
+        - tensorflow-hub {{ libver_tensorflow_hub }}
+        # Adding the follwoing temporarily to fix a 
+        # tensorflow-dependency version-mismatch issue.
+        - absl-py {{ "~=0.10" }}
+        - wrapt {{ "~=1.12.1" }}
+      run_constrained:
+        # NOTE: tensorflow-text is not available on conda-forge
+        - tensorflow-text {{ libver_tensorflow_text }}
+    test:
+      imports:
+        - {{ name_ }}
+      # commands:
+      #   - pip check  # [not win]
+      # requires:
+      #   - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+
+  ### transformers_dep
+  {% if use_transformers_dep is defined %}
+  {% if use_transformers_dep or use_all_subpackages %}
+  {% with subpackage_id = "transformers" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      # NOTE: this throws error for windows build.
+      skip: True  # [win]
+      noarch: generic
+      string: {{ build_num }}_{{ subpackage_id }}_extra
+    requirements:
+      run:
+        # RECIPE: 
+        # "transformers": base_packages + transformers_dep
+        - {{ name_ }} {{ version }}
+        - transformers {{ libver_transformers }}
+    test:
+      imports:
+        - {{ name_ }}
+      commands:
+        - pip check  # [not win]
+      requires:
+        - pip
+  {% endwith %}
+  {% endif %}
+  {% endif %}
+
+  # sentence_tfm_dep
+  {% if use_sentence_tfm_dep is defined %}
+  {% if use_sentence_tfm_dep or use_all_subpackages %}
+  {% with subpackage_id = "sentence_tfm" %}
+  - name: {{ name_ }}-{{ subpackage_id }}
+    build:
+      # NOTE: this throws error for windows build.
+      skip: True  # [win]
```

Comment:
Same as in https://github.com/conda-forge/staged-recipes/pull/16963#issuecomment-974666530
```suggestion
```

Reply:
@BastianZim If I don't use this `skip: True  # [win]`, then any subpackage that fails for windows, will also halt builds of any other subpackage (which follows the failing subpackage). If the subpackage after the failing subpackage happens to pass all the CI's, then removing the `skip: True`, will also stop a valid build. 

Isn't it? Or, am I missing something here?

Reply:
If one fails all fail, it doesn't matter what order the packages are in since the packages are only uploaded at the end.

Reply:
Ok.

Reply:
Done.